client/pkg/omnictl/internal/access/client.go

```diff
@@ -159,3 +162,19 @@ If you want to enable the version validation and disable this warning, set githu
 
 	return nil
 }
+
+func checkVersionWarning(sysVersion *system.SysVersion) {
+	backendVersion, err := semver.ParseTolerant(sysVersion.TypedSpec().Value.BackendVersion)
+	if err != nil {
+		return
+	}
+
+	clientVersion, err := semver.ParseTolerant(version.Tag)
+	if err != nil {
+		return
+	}
+
+	if clientVersion.Major != backendVersion.Major || clientVersion.Minor != backendVersion.Minor {
+		fmt.Printf("[WARN] omnictl major or minor version differs from the backend version: %q vs %q.\n", clientVersion.String(), backendVersion.String())
```

Comment:
I feel a simpler message is clearer. Maybe something like:
```suggestion
		fmt.Printf("[WARN] omnictl version differs from the backend version: %q vs %q.\n", clientVersion.String(), backendVersion.String())
```